cmd/application_config_set.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/civo/civogo"
+	"github.com/civo/cli/config"
+	"github.com/civo/cli/utility"
+
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var configName, configValue string
+
+var appConfigSetCmd = &cobra.Command{
+	Use:     "set",
+	Args:    cobra.MinimumNArgs(1),
+	Short:   "Set application config",
+	Example: "civo app config set APP_NAME --name=foo --value=bar",
+	Run: func(cmd *cobra.Command, args []string) {
+		client, err := config.CivoAPIClient()
+		if err != nil {
+			utility.Error("Creating the connection to Civo's API failed with %s", err)
+			os.Exit(1)
+		}
+
+		findApp, err := client.FindApplication(args[0])
+		if err != nil {
+			utility.Error("%s", err)
+			os.Exit(1)
+		}
+
+		updatedConfig := civogo.EnvVar{
+			Name:  configName,
```

Comment:
If configName is "PORT", can you error out with a message: "PORT is an immutable field  picked by Civo"